Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/border.h

```diff
@@ -115,10 +113,10 @@ namespace Polygon_mesh_processing {
 
   /*!
   \ingroup PkgPolygonMeshProcessing
-  * collects the border of a surface patch
-  * defined as a face range. The border is "seen from inside" the patch,
-  * i.e. the collected halfedges are
-  * the ones that belong to the input faces.
+  * collects the border halfedges of a surface patch
+  * defined as a face range, that is the halfedges who either do not have an
+  * incident face or whose incident face does not belong to the patch.
```

Comment:
I don't like much this sentence. I suggest :
"... defined as a face range. For each returned halfedge `h`, `opposite(h, pmesh)` belongs to only one of the input faces."